sql-scripts/adding_from_internal_to_secure_message.sql

```diff
@@ -1,14 +1,14 @@
 -- Script to add a from_internal column to secure_message
 -- and populate from_internal to true
--- for messages that were sent from internal as bres user
+-- for messages that were sent from internal user
 -- Also drops the internal_sent_audit table and actors tables if present
 
 
 Alter table securemessage.secure_message ADD from_internal Bool Default False;
 
 update securemessage.secure_message m
 set from_internal = true
-where m.msg_id in (select s.msg_id from securemessage.status s where s.actor = 'BRES' and s.label='SENT');
+where m.msg_id in (select s.msg_id from securemessage.status s where s.label='SENT');
```

Comment:
This shouldn't be modified, if this was ever run it would now set from_internal to true on all sent messages!

Reply:
Can be removed then, we are not going to have BRES user anymore or we have to create a card to generate a parametric SQL.

Reply:
I don't think this script is actually used anywhere (double check that beforehand though!), it can probably be safely deleted.

Reply:
Agreed, we should delete the file.